docker_daemon/README.md

```diff
@@ -1,12 +1,15 @@
 # Docker_daemon Integration
 {{< img src="integrations/docker/docker.png" alt="Docker default dashboard" responsive="true" popup="true">}}
+
 ## Overview
 
-Get metrics from docker_daemon service in real time to:
+Configure this Agent check to get metrics from docker_daemon service in real time to:
 
 * Visualize and monitor docker_daemon states
 * Be notified about docker_daemon failovers and events.
 
+**Note**: The Docker check has been rewritten in Go for Agent v6 to take advantage of the new internal architecture. [Consult our dedicated agent v6 setup](#agent-v6) for more information.
```

Comment:
I'd add something to explain that the current check is still maintained but only works on Agents <6